src/include/sof/audio/component.h

```diff
@@ -189,11 +190,12 @@ struct comp_ops {
 	void (*free)(struct comp_dev *dev);
 
 	/** set component audio stream parameters */
-	int (*params)(struct comp_dev *dev);
+	int (*params)(struct comp_dev *dev,
+		      struct sof_ipc_stream_params *params);
 
 	/** set component audio stream parameters */
 	int (*dai_config)(struct comp_dev *dev,
-		struct sof_ipc_dai_config *dai_config);
+			  struct sof_ipc_dai_config *dai_config);
```

Comment:
Pass-by coding style change/fix, didn't check if it's a separate commit. Reminding myself to check this once I finish looking at everything else.

Reply:
Ah, it is indeed separate. We're good then.